### ISSUE TITLE
Compatability with sequel-rails gem

### ISSUE THREAD

**Issue by tombruijn** [appsignal/appsignal-ruby#799]
When using sequel-rails, we recommend users to [disable the AppSignal's sequel instrumentation](https://docs.appsignal.com/ruby/integrations/sequel.html#sequel-rails-integration) by setting `instrument_sequel` to `false`. This is because the sequel-rails gem records ActiveSupport::Notification events, and otherwise they would be recorded twice.

The problem is that we have no event formatter for the sequel-rails `sql.sequel` events, so they are recorded without an event body, which means users can't see which query was performed this way.

We should add an event formatter for `sql.sequel` to record the SQL query in the event recorded in AppSignal.

Mentioned in https://app.intercom.com/a/apps/yzor8gyw/inbox/inbox/4356044/conversations/16410700082854

## Example event formatter:

```ruby
module Appsignal
  class EventFormatter
    # @api private
    module Sequel
      # Compatability with the sequel-rails gem.
      # The sequel-rails gem adds its own ActiveSupport::Notifications events
      # that conflict with our own sequel instrumentor. Without this event
      # formatter the sequel-rails events are recorded without the SQL query
      # that's being executed.
      class SqlFormatter
        def format(payload)
          [payload[:name].to_s, payload[:sql], SQL_BODY_FORMAT]
        end
      end
    end
  end
end

Appsignal::EventFormatter.register(
  "sql.sequel",
  Appsignal::EventFormatter::Sequel::SqlFormatter
)
```

**Comment by Aryk** [comment]
For me, I needed to nil the first element:

```ruby
 def format(payload)
          [nil, payload[:sql], SQL_BODY_FORMAT]
        end
```

**Comment by tombruijn** [comment]
@Aryk You probably got an error about an argument being a class type instead of a String? That's been fixed in PR #801.

**Comment by Aryk** [comment]
Yeah I got something like that...looking forward to the gem update

**Comment by tombruijn** [comment]
@Aryk The fix for this has been released in Ruby gem 3.0.19.
